Tasks/Maven/maventask.ts

```diff
@@ -212,11 +213,17 @@ function createMavenSQRunner(sqHostUrl, sqHostUsername, sqHostPassword, sqDbUrl?
 function applyPmdGoals(mvnRun: trm.ToolRunner):void {
     if (tl.getInput('pmdAnalysisEnabled', true) != 'true') {
         console.log('PMD analysis is not enabled');
+        return;
     }
     console.log('PMD analysis is enabled');
     pmd.applyPmdArgs(mvnb);
 }
 
+// Take a result object and create one line of the build summary from it.
+function buildSummaryLineFromResult(analysisResult:ar.AnalysisResult):string {
+    return `${analysisResult.toolName} found ${analysisResult.totalViolations} violations in ${analysisResult.filesWithViolations} files.`;
```

Comment:
You might want to make the message prettier in case you find 0 violations, you can simply say "PMD found no violations".